charts/keycloak/README.md

```diff
@@ -377,6 +379,33 @@ The headless service that governs the StatefulSet is used for DNS discovery.
 
 ## Upgrading
 
+### From chart versions < 7.0.0
+Version 7.0.0 is a minor update - but it does break backwards-compatibility with the existing
```

Comment:
I wouldn't call it a minor update. We do bump the major version. Just leave that out.